python/ray/state.py

```diff
@@ -383,9 +383,35 @@ def actor_table(self, actor_id=None):
                     ray.ActorID(actor_id_binary))
             return results
 
-    def client_table(self):
-        """Fetch and parse the Redis DB client table.
+    def node_table(self):
```

Comment:
node_table and client_table is same?

Reply:
Mark client table as deprecated and remove it when all places use node_table.